channel2/cmd/channel2/subcmd/underlay/listener.go

```diff
@@ -61,6 +61,8 @@ func runListener(_ *cobra.Command, _ []string) {
 	switch listenerUnderlay {
 	case "classic":
 		listener = channel2.NewClassicListener(id, endpoint, channel2.DefaultConnectOptions())
+	case "wss":
+		listener = channel2.NewWssListener(id, endpoint, channel2.DefaultConnectOptions(), "serverCert-here", "key-here")
```

Comment:
The `id` field here is an `Identity. Identity` which has the controller/router's identity inside of it. At this level in the code, that is a server cert/private key.

The `serverCert-here` and the `key-here` seem odd with that in mind. I'd expect this listener to use the same `Identity.Identity` as the other listeners and follow the same pattern.

Reply:
this was obviated by the new wss transport, so it has been removed.